crates/sncast/tests/data/scripts/state_script/scripts/src/state_script.cairo

```diff
@@ -0,0 +1,40 @@
+use sncast_std::{
+    declare, deploy, invoke, call, DeclareResult, DeployResult, InvokeResult, CallResult, get_nonce
+};
+
+fn main() {
+    let max_fee = 99999999999999999;
+    let salt = 0x5;
+
+    let declare_nonce = get_nonce('latest');
```

Comment:
What's the rationale for using `latest` instead of `pending` here? I would expect that it might be the root issue of the failing test.

Reply:
this may be valid for tests on real nodes. If I remember correctly atm devnet just returns the same nonce for pending and latest, so this does not really make much difference for devnet tests (as blocks are being accepted instantly).